AkitaInteractive/Hachiko-Engine
# Meta files standarization

Please guys, with this pr the resource manager changes a lot with all the new Meta file structure. If you can don't just review the code make some QA and test if there is something that doesn't work as it supouse to. Thanks!

Test cases:
- Delete library
- Import of new assets
- Change textures in materials
- Create scenes

Metas should be forever when you include an asset its meta should be also uploaded, so the relation UID asset is never lost. Don't delete metas, if using saved scenes.

Please if there any bug send us the steps to reproduce it. Thanks!
